overlord/hookstate/ctlcmd/refresh.go

```diff
@@ -241,6 +244,24 @@ func (c *refreshCommand) proceed() error {
 	ctx.Lock()
 	defer ctx.Unlock()
 
+	// running outside of hook
+	if ctx.IsEphemeral() {
+		st := ctx.State()
```

Comment:
maybe we want a TODO for now here:

// TODO: consider having a permission via an interface for this before making this not experimental

Reply:
Added.